### PR TITLE
add criminal record check field to interpreter inputs

### PR DESCRIPTION
**Pull request by michel-fresh**
forgot to get this change in as well

### CODE REVIEW

File: frontend/src/components/form/inputs/Interpreter.tsx [hunk 1]
```diff
@@ -111,6 +111,7 @@ export default function InterpreterInputs () {
 
       <StyledField name='supplier' label='Supplier #' />
       <StyledField name='gst' label='GST' />
+      <StyledField name='criminalRecordCheck' label='Recrod Check Received' />
```
Comment (will-freshworks) [inline frontend/src/components/form/inputs/Interpreter.tsx:114]: typo

Comment (will-freshworks) [reply frontend/src/components/form/inputs/Interpreter.tsx:114]: `Recrod`

Comment (will-freshworks) [reply frontend/src/components/form/inputs/Interpreter.tsx:114]: Will this auto-type it to date? Or is it a text field?

**Review by jerryfreshworks — APPROVED** [review]

**Review by will-freshworks — CHANGES_REQUESTED** [review]

File: frontend/src/components/form/inputs/Interpreter.tsx (hunk 1 above)
Comment (michel-fresh) [reply frontend/src/components/form/inputs/Interpreter.tsx:114]: it is a text field as they gave us string dates that have no format

**Pull request merged**